modules/ding_library/ding_library.module

```diff
@@ -202,10 +202,15 @@ function ding_library_form_ding_library_node_form_alter(&$form, &$form_state) {
 }
 
 /**
- * After build method. Take out path auto option for ding libraries
+ * After build method. Add javascript to simulate disabled checkbox
```

Comment:
This is a really hacky solution - and not that secure either. Based on the comments I agree about the method.

I would appreciate a comment in the code explaining why this approach is necessary.